config/config.go

```diff
@@ -208,7 +208,7 @@ func (ch *ConfigHandler) fetchConfig() error {
 			return fmt.Errorf("failed to generate wg keys: %w", keyErr)
 		}
 
-		if writeErr := os.WriteFile(ch.wgKeyPath, []byte(privateKey.String()), 0o600); writeErr != nil {
+		if writeErr := os.WriteFile(ch.wgKeyPath, []byte(privateKey.String()), 0600); writeErr != nil {
```

Comment:
At least gocritic prefers the new octal literal style introduced in Go 1.13 -- don't think these need to change.